presidio-anonymizer/app.py

```diff
@@ -22,19 +25,38 @@ def __init__(self):
         def anonymize():
             content = request.get_json()
             if not content:
-                return "Invalid request json", 400
+                return ErrorResponse("Invalid request json"), 400
             try:
-                data = AnonymizerRequest(content)
-                text = AnonymizerEngine().anonymize(data)
+                engine = AnonymizerEngine()
```

Comment:
since you already refactored, can this be a singleton in a class field., not to instantiate each time?

Reply:
I am strongly opposing any singleton pattern if absolutely no need.
Please explain.

Reply:
you instantiate AnonymizerEngine() on each call, when it can be done once.